linux/deploy_vm/flatcar/install_python3.sh

```diff
@@ -78,6 +78,9 @@ else
     echo "Remove directory $ap_python3_download_dir"
     rm -rf $ap_python3_download_dir
 
+    # Add /opt/bin to PATH for python auto discovery
```

Comment:
Then no need to set Python path in this file `flatcar_post_config.yml`.

Reply:
I would prefer to keep it there in case user use other python path like /usr/bin/python, which doesn't exists. And for existing VM, only vm_python=auto can work, otherwise, incorrect python path still leads to failure.

Reply:
Here if the installed Python in the new deployed VM is "/opt/bin/python", we can set it if it's not before executing "update_inventory", no need to execute "update_inventory" firstly, then execute "add_host" again, since we know the path at first. As to existing VM, I think we can add this configure later.
```
- block:
    # Overwrite vm_python in inventory
    - name: "Reset vm_python to /opt/bin/python"
      set_fact:
        vm_python: "/opt/bin/python"

    - name: "Update host variable ansible_python_interpreter"
      add_host:
        hostname: "{{ vm_guest_ip }}"
        groups: "target_vm"
        ansible_python_interpreter: "{{ vm_python }}"
      when:
        - vm_guest_ip is defined
        - groups.target_vm is defined
        - vm_guest_ip in groups.target_vm
  when: (vm_python != "/opt/bin/python") or (vm_python != "/opt/bin/python3")
```